go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/sigstore/rekor
 
-go 1.18
+go 1.20
```

Comment:
What's the reasoning for making this the minimum version?

Reply:
Are we using this as a library in other places?
Do you want to bump it at least to 1.19? 1.18 will be deprecated soon

Reply:
I'd keep this at the lowest level possible, unless we need to use features from 1.20 that are deemed necessary. That way we are interoperable with the widest set of other projects (who compile us in via dependency). 

I just looked at https://go.dev/doc/devel/release - since 1.18 is out of support I'd vote yes to moving to 1.19

Reply:
pushed and set to 1.19